core/vm/contracts.go

```diff
@@ -1399,7 +1399,8 @@ func (c *verifyDoubleSignEvidence) RequiredGas(input []byte) uint64 {
 }
 
 var (
-	extraSeal = 65
+	extraSeal  = 65
+	maxUnit256 = new(big.Int).SetUint64(0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff)
```

Comment:
`0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff`, but the function here is `SetUint64`.

I will suggest just set the max  uint64 not uint256

Reply:
try this, new(big.Int).SetBytes([]byte{
		0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
		0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
		0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
		0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
	}).Bytes()

Reply:
Done